config/deploy/staging.rb

```diff
@@ -9,6 +9,10 @@
 server "fpul-staging1", user: "deploy", roles: %w[app db web drupal_primary]
 server "fpul-staging2", user: "deploy", roles: %w[app db web drupal_secondary]
 
+set :drupal_fileshare_mount, '/mnt/nfs/drupal7'
+
+set :files_dir, 'fpul_staging_files'
```

Comment:
someone will need to make a call on how to actually create and share these.

Reply:
We need to manually move the files from local file system on the app server where we they live now. This directory is somewhat overloaded as it contains both content (images, pdfs that users have embedded in webpages) and transient data like compressed web asset files that Drupal produces and deletes on it's own.

Reply:
We can take them from the current production server.